src/test/java/org/prebid/server/bidder/yieldlab/YieldlabBidderTest.java

```diff
@@ -111,6 +111,59 @@ public void makeHttpRequestsShouldSendRequestToModifiedUrlWithHeaders() {
                         tuple("Cookie", "id=buyeruid"));
     }
 
+    @Test
+    public void constructExtImpShouldWorkWithDuplicateKeysTargeting() {
+        // given
+        final Map<String, String> targeting = new HashMap<>();
+        targeting.put("key1", "value1");
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(List.of(
+                        Imp.builder()
+                                .banner(Banner.builder().w(1).h(1).build())
+                                .ext(mapper.valueToTree(ExtPrebid.of(null,
+                                        ExtImpYieldlab.builder()
+                                                .adslotId("1")
+                                                .supplyId("2")
+                                                .adSize("adSize")
+                                                .targeting(targeting)
+                                                .extId("extId")
+                                                .build())))
+                                .build(),
+                        Imp.builder()
+                                .banner(Banner.builder().w(1).h(1).build())
+                                .ext(mapper.valueToTree(ExtPrebid.of(null,
+                                        ExtImpYieldlab.builder()
+                                                .adslotId("2")
+                                                .supplyId("2")
+                                                .adSize("adSize")
+                                                .targeting(targeting)
+                                                .extId("extId")
+                                                .build())))
+                                .build()
+                        )
+                )
+                .device(Device.builder().ip("ip").ua("Agent").language("fr").devicetype(1).build())
+                .regs(Regs.of(1, ExtRegs.of(1, "usPrivacy")))
+                .user(User.builder().buyeruid("buyeruid").ext(ExtUser.builder().consent("consent").build()).build())
+                .site(Site.builder().page("http://www.example.com").build())
+                .build();
+
+        // when
+        final Result<List<HttpRequest<Void>>> result = yieldlabBidder.makeHttpRequests(bidRequest);
+
+
```

Comment:
Our codestyle plugin is very angry ;)
`There is more than 1 empty line one after another. [EmptyLineSeparator]`

Reply:
Let's give him what he wants :joy: